android/app/src/main/java/com/expensify/chat/MainApplication.kt

```diff
@@ -0,0 +1,77 @@
+package com.expensify.chat
+
+import android.content.res.Configuration
+import android.database.CursorWindow
+import androidx.multidex.MultiDexApplication
+import com.expensify.chat.bootsplash.BootSplashPackage
+import com.facebook.react.PackageList
+import com.facebook.react.ReactApplication
+import com.facebook.react.ReactNativeHost
+import com.facebook.react.ReactPackage
+import com.facebook.react.defaults.DefaultNewArchitectureEntryPoint.load
+import com.facebook.react.defaults.DefaultReactNativeHost
+import com.facebook.react.modules.i18nmanager.I18nUtil
+import com.facebook.soloader.SoLoader
+import com.google.firebase.crashlytics.FirebaseCrashlytics
+import expo.modules.ApplicationLifecycleDispatcher
+import expo.modules.ReactNativeHostWrapper
+
+class MainApplication : MultiDexApplication(), ReactApplication {
+    override val reactNativeHost: ReactNativeHost = ReactNativeHostWrapper(this, object : DefaultReactNativeHost(this) {
+        override fun getUseDeveloperSupport() = BuildConfig.DEBUG
+
+        override fun getPackages(): List<ReactPackage> {
+            val packages: MutableList<ReactPackage> = PackageList(this).packages
+            // Packages that cannot be autolinked yet can be added manually here, for example:
+            // packages.add(new MyReactNativePackage());
+            packages.add(BootSplashPackage())
+            packages.add(ExpensifyAppPackage())
+            packages.add(RNTextInputResetPackage())
+            return packages
+        }
```

Comment:
```suggestion
        override fun getPackages(): List<ReactPackage>  = 
            PackageList(this).packages.apply {
            // Packages that cannot be autolinked yet can be added manually here, for example:
            // add(MyReactNativePackage());
            add(BootSplashPackage())
            add(ExpensifyAppPackage())
            add(RNTextInputResetPackage())
        }
```

Maybe this is used in 0.73.2. (Referencing from upgrade helper.